repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
Shouldn't the versioning of the repository.rdf resource mitigate any potential backwards incompatibility:
http://fedora.info/definitions/v4/2016/03/01/repository